tests/test_phase_psf.py

```diff
@@ -1378,6 +1377,73 @@ def test_pickle():
         np.testing.assert_equal(wf, wf3)
 
 
+@timer
+def test_table_screen():
+    # Mimic an OpticalScreen using a TableScreen.
+    rng = galsim.UniformDeviate(57721)
+    for _ in range(10):
+        aberrations = np.empty(7)
+        rng.generate(aberrations)
+        aberrations -= 0.5
+        aberrations *= 0.1
+        aberrations[0:2] = 0.0
+        Zscreen = galsim.OpticalScreen(
+            diam=4.0,
+            aberrations=aberrations,
+            lam_0=750.0
+        )
+
+        u1 = np.linspace(-2, 2, 200)
+        u, v = np.meshgrid(u1, u1)
+        wf = Zscreen.wavefront(u, v)
+
+        table = galsim.LookupTable2D(u1, u1, wf.T, interpolant='spline')
```

Comment:
Note the `wf.T` here.  LUT2D and `screen.wavefront()` appear to be inconsistent with respect to x/y axes ordering.

Reply:
I feel like this is a sign of a bug somewhere. Because AtmosphericScreen even uses a LookupTable2D in places for doing the wavefront and gradient. But those start with a random screen, so I don't think we could have noticed a transposition error if there was one there.

I think we should have the output from wavefront of an OpticalScreen be a valid input into this Table and be consistent. I'll see if I can track down where the root of the disagreement is happening.

Reply:
Tests of the Table2D functionality have code like this:
```
    x = np.linspace(0.1, 3.3, 25)
    y = np.linspace(0.2, 10.4, 75)
    yy, xx = np.meshgrid(y, x)  # Note the ordering of both input and output here!
    z = f(xx, yy)
```
whereas phase screens seem to have the standard order
```
    fx, fy = np.meshgrid(fx, fx)
```
So that's probably the source of the disagreement.  I think maybe Table2D is the problem here.  Was there some reason that it needed to be defined with the reverse convention?

Reply:
It's possible at the time I had just read [this](https://github.com/GalSim-developers/GalSim/blob/releases/2.2/galsim/image.py#L55), and thought I was being consistent with that.  TBH, I've never fully grokked the numpy/fits/meshgrid/LUT2D/other indexing conventions, I just know that they're not all the same.  Probably time to bite the bullet and pay attention to this.